engine/src/main/java/org/camunda/bpm/engine/impl/cmd/DeleteAttachmentCmd.java

```diff
@@ -16,75 +16,80 @@
  */
 package org.camunda.bpm.engine.impl.cmd;
 
-import static org.camunda.bpm.engine.impl.util.EnsureUtil.ensureNotNull;
-
-import java.io.Serializable;
-
+import org.apache.commons.lang3.StringUtils;
```

Comment:
There seems to be some autoformatting happening on my end - is this reformatting aligned with your styleguide or should I adjust my settings?